### PR TITLE
initial commit added create-react-app

### PR DESCRIPTION
**Pull request by kareemuddinmasir**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import { BrowserRouter as Router, Route, } from "react-router-dom";
+
+class App extends Component {
+  render() {
+    return (
+      <Router>
+      <div className="App">
+      <Route exact path='/' component={Home}/>
+      <Route exact path='/SignIn' component = {SignIn}/>
```
Comment (naquiuddin) [inline src/App.js]: paths should be in small letters, change `/SignIn` to `/sign-in`

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      <Route  path='/SignUp' component = {SignUp}/>
```
Comment (naquiuddin) [inline src/App.js]: Same here. Use lowercase letters for url paths.

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      <Route  path='/Post' component = {Post}/>
```
Comment (naquiuddin) [inline src/App.js]: same here as well.

File: src/Home.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+//import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+//import { Link } from 'react-router-dom'
```
Comment (naquiuddin) [inline src/Home.js]: Remove these comments. Unnecessary comments should not be used.

File: src/SignIn.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+//import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+//import { Link } from 'react-router-dom'
```
Comment (naquiuddin) [inline src/SignIn.js]: Remove the comments here as well.

File: src/SignUp.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+//import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+//import { Link } from 'react-router-dom'
```
Comment (naquiuddin) [inline src/SignUp.js]: Same here. These comments are not required.

File: src/App.js (hunk 1 above)
Comment (kareemuddinmasir) [reply src/App.js]: Done.

File: src/App.js (hunk 2 above)
Comment (kareemuddinmasir) [reply src/App.js]: Changes made.

File: src/App.js (hunk 3 above)
Comment (kareemuddinmasir) [reply src/App.js]: done

File: src/Home.js (hunk 4 above)
Comment (kareemuddinmasir) [reply src/Home.js]: OK. Removed the unnecessary comments.

File: src/SignUp.js (hunk 6 above)
Comment (kareemuddinmasir) [reply src/SignUp.js]: Done .

**Pull request merged**